gnome/spice-gtk/Portfile

```diff
@@ -23,23 +24,27 @@ long_description    gtk-vnc is a VNC viewer widget for GTK. It is built using \
 homepage            https://www.spice-space.org/page/Spice-Gtk
 master_sites        https://www.spice-space.org/download/gtk/
 
-use_bzip2           yes
+use_xz              yes
 
-checksums           rmd160  fb6644c0c70af290929329382f317ab5784856cc \
-                    sha256  b4e6073de5125e2bdecdf1fbe7c9e8c4cabe9c85518889b42f72bf63c8ab9e86 \
-                    size    1437231
+checksums           rmd160  a9e44097fa0b7cc0b658e39a879fec205d9288f2 \
+                    sha256  5ae974731baf2b41316d4f0b3ae0c2e47f00bff91a5a617e189cd3dedcd96d8e \
+                    size    796112
+
+patchfiles          no-werror.diff
 
 depends_build-append \
                     port:pkgconfig \
+                    port:py38-six port:py38-parsing \
```

Comment:
Does this port use python? I don't see anything else in the portfile referencing any python, so how would the build system know to use this version of python?

Reply:
The meson build system is written in Python; I believe that's where it gets it from. Configuration failed without these.